packages/wonder-blocks-button/components/button.js

```diff
@@ -297,6 +297,7 @@ export default class Button extends React.Component<Props> {
                     spinner={spinner || state.waiting}
                     skipClientNav={skipClientNav}
                     href={href}
+                    target={target}
```

Comment:
What's the reason for adding `target` to button? I think we should make the distinction whether we use `Link` vs `<button />`.

Reply:
To expand a bit, I think that button should use `formtarget` internally.

Reply:
It's only mean to be used when `href` is set.  In that situation `<Button>` will render as either a `<a>` or a `<Link>`.  I haven't enforced this constraint though.

Reply:
Here's the comment from the prop:
```
    /**
     * A target destination window for a link to open in. Should only be used
     * when `href` is specified.
     */
    target?: string,
```

Reply:
Ohh that makes sense, thanks!